src/vizier/services/agent/pem/pem_manager_test.cc

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2018- The Pixie Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gtest/gtest.h>
+
+#include "src/common/testing/testing.h"
+#include "src/vizier/services/agent/pem/pem_manager.h"
+
+DECLARE_bool(disable_SSL);
+
+namespace px {
+namespace vizier {
+namespace agent {
+
+class PEMManagerTest : public ::testing::Test {
+ protected:
+  // void TearDown() override { dispatcher_->Exit(); }
```

Comment:
@benkilimnik did you intend to have this code uncommented or can this be removed?

Reply:
Just removed